yax/core/src/test/scala/doobie/syntax/catchable.scala

```diff
@@ -23,16 +23,13 @@ object catchablespec extends Specification {
       true
     }
 
-    "work on unaliased IConnection" in {
+// TODO Remove yax (https://github.com/tpolecat/doobie/issues/369)
 #+scalaz
+    "work on unaliased IConnection" in {
       42.pure[ConnectionIO].map(_ + 1).attempt
-#-scalaz
-#+cats
-      // TODO Remove yax (https://github.com/tpolecat/doobie/issues/369)
-      (42.pure[ConnectionIO].map(_ + 1): ConnectionIO[Int]).attempt
```

Comment:
Replaced this one with cross paths, because this workaround makes it the same as the aliased one.